src/test/regress/expected/multi_utilities.out

```diff
@@ -210,6 +210,45 @@ begin
     extract(epoch from clock_timestamp() - start_time);
 end
 $$ language plpgsql;
+\c - - - :worker_2_port
```

Comment:
At some point it's probably worth checking whether PostgreSQL changes their own way of doing this; this logic was copied from their own tests.

Reply:
Postgres keeps using the same logic. When I check the runtime of tests, this add +20sec (10 for each worker) since there is no analyze run in workers. Should we decrease the time or is there any other way you can think about?

Reply:
I think it's safest to just do what PostgreSQL does for this; it's too fiddly for us to try to re-solve on our own.